rocky/rocky/views/task_detail.py

```diff
@@ -0,0 +1,79 @@
+import uuid
+
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+from django.views.generic import TemplateView
+from katalogus.views.mixins import BoefjeMixin, NormalizerMixin
+
+from rocky.scheduler import client
+from rocky.views.mixins import OctopoesView
+
+
+class TaskDetailView(OctopoesView, TemplateView):
+    @staticmethod
+    def get_task(task_id):
+        return client.get_task_details(task_id)
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        context["task_id"] = uuid.UUID(kwargs["task_id"]).hex
+        context["task"] = self.get_task(context["task_id"])
+        return context
+
+
+class BoefjeTaskDetailView(BoefjeMixin, TaskDetailView):
+    template_name = "tasks/boefje_task_detail.html"
+    plugin_type = "boefje"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        context["breadcrumbs"] = [
+            {
+                "url": reverse("task_list", kwargs={"organization_code": self.organization.code}),
+                "text": _("Tasks"),
+            },
+            {
+                "url": reverse(
+                    "boefje_task_view",
+                    kwargs={"organization_code": self.organization.code, "task_id": uuid.UUID(context["task"].id).hex},
+                ),
+                "text": context["task"].p_item.data.boefje.id,
+            },
+        ]
+
+        return context
+
+
+class NormalizerTaskDetailView(NormalizerMixin, TaskDetailView):
+    template_name = "tasks/normalizer_task_detail.html"
+    plugin_type = "normalizer"
+
+    def get_output_oois(self, task):
+        try:
+            return self.octopoes_api_connector.list_origins(
+                valid_time=task.p_item.data.raw_data.boefje_meta.ended_at, task_id=uuid.UUID(task.id).hex
```

Comment:
See my previous point about reusing vars

Reply:
How can I reuse the context variable here? Or should I store the value in another way so I'm able to access it here as well?